lib/active_admin/helpers/collection.rb

```diff
@@ -3,13 +3,13 @@ module Helpers
     module Collection
       # 1. removes `select` and `order` to prevent invalid SQL
       # 2. correctly handles the Hash returned when `group by` is used
-      def collection_size(c = collection)
+      def collection_size(collection)
```

Comment:
Unused method argument - `collection`. If it's necessary, use `_` or `_collection` as an argument name to indicate that it won't be used. You can also write as `collection_size(*)` if you want the method to accept any arguments but don't care about them.